ansible/vagrant/.gitignore

```diff
@@ -1 +1,2 @@
+/.vagrant/
```

Comment:
I use leading slash to explicitly state that I'm ignoring `.vagrant` sub-directory in *this* directory.
If I'd use `.vagrant/` git will ignore `.vagrant` sub-directory in *this* directory and *all sub-directories*, which is not what I want, since we don't expect to see `.vagrant` in subdirectories.

@eparis does it make sense or I missing something?